File: go/01-Go/01-Programming-basics/03-Age-in-days/age/age_in_days.go
package age

import (
	"time"

	"github.com/rickb777/date"
)

func AgeInDays(day, month, year int) date.PeriodOfDays {
	birthDate := date.New(year, time.Month(month), day).DaysSinceEpoch()
	today := date.Today().DaysSinceEpoch()

	return today - birthDate
}

File: ruby/01-Ruby/01-Programming-basics/03-Age-in-days/lib/age_in_days.rb
# This "require" line loads the contents of the "date" file from the standard
# Ruby library, giving you access to the Date class.
require "date"

# This is a comment to remind you not to push too early dumbass

def age_in_days(day, month, year)
  # TODO: return the age expressed in days given the day, month, and year of birth
  Date.today.mjd - Date.new(year, month, day).mjd
end
